data/json/items/armor/torso_clothes.json

```diff
@@ -115,16 +121,10 @@
       { "covers": [ "torso" ], "specifically_covers": [ "torso_waist" ], "coverage": 80, "encumbrance": [ 0, 0 ] },
       {
         "covers": [ "leg_l", "leg_r" ],
-        "coverage": 70,
-        "specifically_covers": [ "leg_hip_l", "leg_hip_r" ],
+        "coverage": 22,
+        "specifically_covers": [ "leg_draped_l", "leg_draped_r" ],
```

Comment:
I'd remove leg coverage entirely.  This is like a servers apron in length but missing the pockets.

Reply:
Googling around on this the items I see do appear to cover the top of the leg, even the shortest ones
like this https://www.webstaurantstore.com/choice-black-standard-waist-apron-with-3-pockets-12l-x-26w/160WAISTFHBK.html?utm_source=google&utm_medium=cpc&utm_campaign=GoogleShopping&gclid=CjwKCAjw7eSZBhB8EiwA60kCW5in744IGX9fbKHaZmrNeTDs1S9AwnDjZoYfDs3_zqTXpYFwvkLl1hoCyE4QAvD_BwE